# Define the goal
string hopper_or_excavator  
---
# Define the result
# See COMMON_NAMES::NAV_VISION_RESULT for possible values
uint32 result
---
# Define a feedback message
bool process